### PR TITLE
branding changes

### PR DESCRIPTION
**Pull request by jwgulley**
These are changes for making updates to the branding as described in TRLST-169.  Tipu has kindly reviewed the result and declared himself happy with it..  This PR is for review only at this stage, but with a view to it being approved and applied when the branding update is required.

Known Issues

These changes do not include:
- updates to any documents - e.g. letter of authority templates or email templates;
- updates to the database - see the ticket comment for details, also the fixture changes may be relevant.

### CODE REVIEW

**Review by markhigham — APPROVED** [review]
LGTM. You could've potentially used those partial templates where you've made text changes. Or perhaps added them to context processor and used {{ TRID_full_name }}

File: trade_remedies_public/templates/partials/TRID_abbreviation.html [hunk 1]
```diff
@@ -1 +1 @@
-TRID
\ No newline at end of file
+TRA
```
Comment (davecharles) [inline trade_remedies_public/templates/partials/TRID_abbreviation.html:1]: Do we still need this file?

File: trade_remedies_public/templates/partials/TRID_full_name.html [hunk 2]
```diff
@@ -1 +1 @@
-Trade Remedies Investigations Directorate
\ No newline at end of file
+Trade Remedies Authority
```
Comment (davecharles) [inline trade_remedies_public/templates/partials/TRID_full_name.html:1]: Do we still need this file?

**Review by davecharles — CHANGES_REQUESTED** [review]
I think these templates have been missed:
```
trade_remedies_public/templates/cases/submissions/questionnaire/tasklist.html
trade_remedies_public/templates/cases/submissions/invite/tasklist.html
trade_remedies_public/templates/cases/submissions/invite/submit.html
```

**Comment by Luisella21** [comment]
Do we need to add ORGANISATION_INITIALISM and ORGANISATION_NAME to the env file?

**Comment by jwgulley** [comment]
> I think these templates have been missed:
> 
> ```
> trade_remedies_public/templates/cases/submissions/questionnaire/tasklist.html
> trade_remedies_public/templates/cases/submissions/invite/tasklist.html
> trade_remedies_public/templates/cases/submissions/invite/submit.html
> ```
Thanks, David.

**Comment by jwgulley** [comment]
> Do we need to add ORGANISATION_INITIALISM and ORGANISATION_NAME to the env file?
Thank-you - adding them to local.env.example.

**Review by davecharles — APPROVED** [review]
LGTM

File: trade_remedies_public/core/templatetags/organisation_initialism.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+from core.templatetags import register
+from django.conf import settings
+
+"""
+Template tag to display organisation name and organisation initialism
```
Comment (rossmiller) [inline trade_remedies_public/core/templatetags/organisation_initialism.py:5]: Please change text to "Template tag to display organisation initialism"

File: trade_remedies_public/core/tests.py [hunk 4]
```diff
@@ -29,7 +30,11 @@ def _get_docs():
                 "virus_scanned_at": "2020-05-14T10:16:40+0000",
                 "parent_id": None,
                 "checksum": "207b2135a75c97ed4c16368868d66cfd-1",
-                "type": {"id": 1, "name": "TRID Document", "key": "caseworker"},
+                "type": {
+                    "id": 1,
+                    "name": settings.ORGANISATION_INITIALISM + " Document",
```
Comment (rossmiller) [inline trade_remedies_public/core/tests.py:35]: Please use a f-string here rather than +

**Review by rossmiller — CHANGES_REQUESTED** [review]
Please address the other comments and update the readme with instructions for the new env vars.

File: trade_remedies_public/core/tests.py (hunk 4 above)
Comment (jwgulley) [reply trade_remedies_public/core/tests.py]: Thanks - done.

File: local.env.example [hunk 5]
```diff
@@ -19,3 +19,6 @@ S3_STORAGE_KEY=xx
 S3_STORAGE_SECRET=xx
 AWS_REGION=xx
 
+# Parameters for branding
```
Comment (rossmiller) [inline local.env.example:22]: Change to:
```
ORGANISATION_NAME = "Organisation name placeholder"
ORGANISATION_INITIALISM = "PLACEHOLDER"
```

Comment (jwgulley) [reply local.env.example:22]: done

File: trade_remedies_public/core/templatetags/organisation_initialism.py (hunk 3 above)
Comment (jwgulley) [reply trade_remedies_public/core/templatetags/organisation_initialism.py]: done

**Review by rossmiller — APPROVED** [review]
LGTM!

**Pull request merged**